lib/gantree/version.rb

```diff
@@ -1,3 +1,3 @@
 module Gantree
-  VERSION = "0.6.13"
+  VERSION = "0.6.14"
```

Comment:
You set version 0.6.14 here but on the documentation you set 0.6.16?

Can we get them to line up?